### PR TITLE
Ac 327 split out content

### PR DESCRIPTION
**Pull request by ru-fu**
There are a few "tbd" in the index file that need to be filled in when publishing.

### CODE REVIEW

**Review by morphis — APPROVED** [review]
LGTM

File: backup-and-restore.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+TARBALL_FILE=$(basename $(find ./ -name *.tar.bz2))
+ # Upload the tarball to public or private cloud storage service
+curl -i -X POST --data-binary @"${TARBALL_FILE}" <cloud_storage_upload_url>
+```
+Running this script in an [addon backup hook](https://discourse.ubuntu.com/t/managing-addons/17759#heading--backup-and-restore) will back up the user data of a particular application with `aam` and upload the resulting tarball file to the cloud storage service when a container is stopped.
+
+If [`boot-package`](https://discourse.ubuntu.com/t/managing-applications/17760#heading--create-applications) can be retrieved from the apk file or is specified in the application manifest file, you can also back up the boot application data simply with the flag `--boot-package`.
+
+```bash
+$ aam backup --boot-package
```
Comment (aguadoenzo) [inline backup-and-restore.md:19]: Are we removing leading `$` here or are you going to do a separate PR for this ?

File: backup-and-restore.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Or by relying on the boot package of the container:
+
+```bash
+$ aam restore -p app-data.tar.bz2 --boot-package
+```
+
+## Filter data to be backed up
+
+Sometimes, not every piece of data is useful (for example, cache), and backing up the entire application data takes long time and occupies more disk space if the application data is large. `aam` supports two filters to back up files that match wildcard patterns:
```
Comment (aguadoenzo) [inline backup-and-restore.md:44]:
```suggestion
Sometimes, not every piece of data is useful (for example, cache), and backing up the entire application data takes a long time and occupies more disk space if the application data is large. `aam` supports two filters to back up files that match wildcard patterns:
```

File: geographic-location.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+Anbox Cloud allows specifying a static geographic location which is provided to the Android container.  The location is configured in the file `/var/lib/anbox/static_gps_position` within the Anbox container.
```
Comment (aguadoenzo) [inline geographic-location.md:1]: @adglkh didn't you add an API in Anbox to allow changing that dynamically ?

File: index.md [hunk 4]
```diff
@@ -163,4 +166,12 @@ Check the [release notes](https://discourse.ubuntu.com/t/release-notes/17842) fo
...
 |https://discourse.ubuntu.com/t/integrate-a-client-side-virtual-keyboard/23643|/docs/integrate-keyboard|
+|https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750|/docs/howto/upgrade/upgrade-anbox|
+|tbd|/docs/howto/upgrade/upgrade-appliance|
+|https://discourse.ubuntu.com/t/usecase-container-configuration/17782|/docs/howto/containers/geographic-location|
+|tbd|/docs/howto/containers/backup-and-restore|
+|https://discourse.ubuntu.com/t/managing-images/17758|/docs/howto/manage/images|
+|tbd|/docs/ref/provided-images|
 [/details]
+
+
```
Comment (aguadoenzo) [inline index.md:177]:
```suggestion
```

File: upgrade-anbox.md [hunk 5]
```diff
@@ -8,46 +8,13 @@ The upgrade instructions detail the revisions each charm needs to be upgraded to
...
-
-## Juju based deployments
-
-The following sections describe how to update a Juju based deployment of Anbox Cloud.
-
 <a name="juju-version"></a>
-### Juju Version
+## Check Juju version
 
 With the 1.8 release of Anbox Cloud you **MUST** use Juju >= 2.8. If your deployment doesn't yet use Juju 2.8 you have to upgrade your controller and all models first. See the [Juju documentation](https://juju.is/docs/upgrading-models) for more details on how to upgrade the Juju controller and all models to Juju 2.8.
```
Comment (aguadoenzo) [inline upgrade-anbox.md:19]:
```suggestion
With the 1.8 release of Anbox Cloud you **MUST** use Juju 2.8. If your deployment doesn't yet use Juju 2.8 you have to upgrade your controller and all models first. See the [Juju documentation](https://juju.is/docs/upgrading-models) for more details on how to upgrade the Juju controller and all models to Juju 2.8.
```
Since we don't support 2.9 yet, we should tag 2.8

File: upgrade-appliance.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    $ sudo apt upgrade
+    
+The Anbox Cloud Appliance includes an `upgrade` command which will perform all relevant upgrade steps to a newer version of the appliance.  First you can check if an update is available:
+
+    $ anbox-cloud-appliance status
+    status: ready
+    update-available: true
+    reboot-needed: false
+
+> **IMPORTANT:** While the upgrade process is active API endpoints and the dashboard will not be available. Anbox containers  will stay active and existing streams will also not be interrupted.
```
Comment (aguadoenzo) [inline upgrade-appliance.md:13]:
```suggestion
> **IMPORTANT:** While the upgrade process is active API endpoints and the dashboard will not be available. Anbox containers will stay active and existing streams will also not be interrupted.
```

**Review by aguadoenzo — CHANGES_REQUESTED** [review]

File: backup-and-restore.md (hunk 1 above)
Comment (ru-fu) [reply backup-and-restore.md]: I'm planning to do that as a global change, but since this is a new file, it makes sense to fix it right away instead of adding it wrong first. Will update.

**Review by ru-fu — COMMENTED** [review]
Updated and included the navigation changes on the index page.

File: geographic-location.md (hunk 3 above)
Comment (adglkh) [reply geographic-location.md:1]: We support specifying static gps data via  `/var/lib/anbox/static_gps_position` from the very beginning of location feature support as described above. Then we add support to dynamically change the location via HTTP API. For the backward compatibility purpose, we need to keep both.  
As @ru-fu created a dedicated page to introduce this feature,  I'd suggest having two sub-headers on this page to make it clear on the way of changing GPS data through:
* static GPS data
* HTTP API

Regarding the GPS data change through HTTP API, we could navigate people to the following section https://github.com/anbox-cloud/discourse-docs/blob/main/anbox-http-api.md#heading--10location

**Comment by adglkh** [comment]
LGTM

**Review by aguadoenzo — APPROVED** [review]
